include/mbedtls/mbedtls_config.h

```diff
@@ -3997,4 +3997,13 @@
  */
 //#define MBEDTLS_P256M_EXAMPLE_DRIVER_ENABLED
 
+
+/**
+ * Uncomment to enable using new bignum code in the ECC modules.
+ *
+ * \warning ECC implementation using new bignum code is a work in progress,
```

Comment:
We've used the work "experimental", which to me implies options that might become standard configuration options but also might not. Since this will eventually become part of the standard code base I see why "work in progress" is used, but this configuration option will hopefully go, so do you think "experimental" (as a term we have used several times with new configuration options) would be better?